debezium-connector-oracle/src/main/java/io/debezium/connector/oracle/logminer/events/Transaction.java

```diff
@@ -81,6 +82,16 @@ public void removeEventWithRowId(String rowId) {
         });
     }
 
+    public String getUserName() {
+        return userName;
+    }
+
+    public void setUserName(String userName) {
```

Comment:
I would prefer we pass `userName` as a constructor argument rather than using a setter method.  The basis for your change is that the "transaction begin" will contain the user name but portions of the transaction that are mined in subsequent mining sessions won't have a username, so it seems fitting to pass this as a constructor argument.